### PR TITLE
Add new ocr engine

### PR DESCRIPTION
**Pull request by wizard1238**

### CODE REVIEW

**Comment by wizard1238** [comment]
changed to new handwriting engine

File: README.md [hunk 1]
```diff
@@ -24,8 +24,13 @@ First set up a MongoDB server (locally or on [MongoDB Atlas](https://www.mongodb
 let mongooseURL = 'my_mongodb_url_with_key'
 export { mongooseURL as mongooseURL }
 ```
+On the server, you also need python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python)  
 Then run the following
 ```bash
+apt install git
```
Comment (TanayB11) [inline README.md]: We don't need to tell people to install git
1. Not everybody uses linux (or uses `apt` as their package manager)
2. Anybody who wants to use the codebase probably already has git

File: README.md [hunk 2]
```diff
@@ -24,8 +24,13 @@ First set up a MongoDB server (locally or on [MongoDB Atlas](https://www.mongodb
 let mongooseURL = 'my_mongodb_url_with_key'
 export { mongooseURL as mongooseURL }
 ```
+On the server, you also need python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python)
```
Comment (TanayB11) [inline README.md]: Replace with
`On the server, make sure you have Python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python) installed.`

File: README.md (hunk 1 above)
Comment (TanayB11) [reply README.md]: Replace this line with a comment: `# Prepare OCR neural net dependencies`

File: README.md [hunk 3]
```diff
@@ -24,8 +24,13 @@ First set up a MongoDB server (locally or on [MongoDB Atlas](https://www.mongodb
 let mongooseURL = 'my_mongodb_url_with_key'
 export { mongooseURL as mongooseURL }
 ```
+On the server, you also need python3 and [TensorFlow Lite](https://www.tensorflow.org/lite/guide/python)  
 Then run the following
 ```bash
+apt install git
+pip3 install svg.path numpy
+pip3 install git+https://github.com/mcyph/cnn_chinese_hw
+
```
Comment (TanayB11) [inline README.md:33]: Add a comment: `# Start up node.js server`

File: server/controllers/ocr-nextController.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const spawn= require('child_process').spawn
+
+exports.ocr = function(req, res, next) {
+    var strokes = req.body.strokes || "[[(75,128),(83,172),(85,196)],[(84,127),(91,124),(123,121),(151,121),(161,121),(161,128),(165,157),(164,181),(162,195)],[(89,187),(103,194),(129,200),(149,199)]]"
```
Comment (TanayB11) [inline server/controllers/ocr-nextController.ts]: Remove `|| randomStrokeArray`, send a `return` instead.

**Review by TanayB11 — CHANGES_REQUESTED** [review]
TODO:
1. Update README.md according to the comments I posted on the review
2. Remove tomoe data (entire data/tomoe directory)
3. Update OCR code according to comments

Some notes: We're going to move the iOS app to a separate repo so I'm expecting to get rid of the logos soon.

Also we'll need to lint the server code sometime.

Otherwise lgtm!

**Pull request closed without merging**